groovy/inductor/inductor_xpu_backend_e2e.groovy

```diff
@@ -0,0 +1,304 @@
+node(env.nodes_label){
+    cleanWs()
+    stage('setup env') {
+        println('================================================================')
+        println('setup nodes env')
+        println('================================================================')
+        sh '''
+        set -e
+        set +x
+        if [ -e ${HOME}/miniconda3/etc/profile.d/conda.sh ];then
+            source ${HOME}/miniconda3/etc/profile.d/conda.sh 2>&1 >> /dev/null
+            if [ -d ${HOME}/miniconda3/envs/${conda_env} ];then
+                conda activate ${conda_env}
+            else
+                conda create -n ${conda_env} cmake ninja pillow python=${python_version}
+            fi
+        else
+            wget -q -e "https_proxy=http://proxy-us.intel.com:912" https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh -b
+            source ${HOME}/miniconda3/etc/profile.d/conda.sh 2>&1 >> /dev/null
+            conda create -n ${conda_env} cmake ninja pillow python=${python_version}
+        fi
+
+        # set gpu governor
+        if [[ -z "${USER_PASS}" ]];then USER_PASS="gta";fi
+        for i in $(seq 0 $(($(nproc)-1)))
+        do
+            echo "${USER_PASS}" | sudo -S sh -c "echo performance > /sys/devices/system/cpu/cpu$i/cpufreq/scaling_governor"
```

Comment:
Such cpu mode can be hardcode in system bashrc or profile, and execute it by default

Reply:
removed